configure.ac

```diff
@@ -3004,7 +3004,7 @@ if test "$enable_f77" = yes ; then
     fi
 
     # Include a defined value for Fint is int
-    if test "$SIZEOF_F77_INTEGER" == "$ac_cv_sizeof_int" ; then
+    if test "$pac_cv_f77_sizeof_integer" == "$ac_cv_sizeof_int" ; then
```

Comment:
`==` is bashism and `=` should be used instead.

Reply:
Thanks for catching it

Reply:
https://github.com/pmodels/mpich/pull/7658